backend/molgenis-emx2-datamodels/src/main/java/org/molgenis/emx2/datamodels/BiobankDirectoryLoader.java

```diff
@@ -1,14 +1,35 @@
 package org.molgenis.emx2.datamodels;
 
+import org.molgenis.emx2.Database;
+import org.molgenis.emx2.Privileges;
 import org.molgenis.emx2.Schema;
 import org.molgenis.emx2.io.MolgenisIO;
+import org.molgenis.emx2.sql.SqlDatabase;
 
 public class BiobankDirectoryLoader extends AbstractDataLoader {
 
+  public static final String ONTOLOGIES = "Ontologies";
```

Comment:
Not a full review, but noticed this name. Isn't 'Ontologies' a bit too broad? Might be safer to call it something like DirectoryOntologies or BiobankOntologies.

Reply:
Looking from a perspective where you have it installed on a dedicated "BBMRI" server not.
However, I do agree that for other purposes where you can have multiple "projects", like f.e. Catalogue, on the same servers it might become confusing, therefore I've adjusted it.